src/plot.jl

```diff
@@ -0,0 +1,31 @@
+using RecipesBase
+
+@recipe function f(ga::GeoArray)
+    xflip --> false
+    yflip --> true
+    aspect_ratio --> 1
+    seriestype := :heatmap
+    color := :deep
+
+    coords = centercoords(ga)
+    x = map(x->x[1], coords[:, 1])
+    y = map(x->x[2], coords[end, :])
+
+    xlims --> (minimum(x), maximum(x))
+    ylims --> (minimum(y), maximum(y))
+
+    z = ga[:,:,1]'  # only first layer for now
```

Comment:
Perhaps we can use a `PermutedDimsArray` here to avoid te extra copy. There is no Plots argument that can just interpret it transposed?